internal/olm/fbcutil/util.go

```diff
@@ -20,14 +20,16 @@ import (
 	"errors"
 	"fmt"
 	"io/ioutil"
+	"log"
 	"os"
 
 	"github.com/operator-framework/operator-registry/alpha/action"
 	"github.com/operator-framework/operator-registry/alpha/declcfg"
 	declarativeconfig "github.com/operator-framework/operator-registry/alpha/declcfg"
 	"github.com/operator-framework/operator-registry/pkg/containertools"
+	"github.com/operator-framework/operator-registry/pkg/image/containerdregistry"
 	registryutil "github.com/operator-framework/operator-sdk/internal/registry"
-	log "github.com/sirupsen/logrus"
+	"github.com/sirupsen/logrus"
```

Comment:
We should not change this. Leave it as `log "github.com/sirupsen/logrus"`

Reply:
Modified.